### PR TITLE
Update find commands to ignore flags

### PR DESCRIPTION
**Pull request by xlzior**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 1]
```diff
@@ -19,7 +20,7 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
+        String trimmedArgs = StringUtil.stripFlags(args.trim());
```
Comment (jiarong15) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:23]: put trim inside the stripFlags function?

**Review by jiarong15 — COMMENTED** [review]
The rest look okay!

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java (hunk 1 above)
Comment (xlzior) [reply src/main/java/seedu/address/logic/parser/FindCommandParser.java:23]: hmm I think the name `stripFlags` is restricted to removing the words that start with `-`, and one would not expect it to also trim the arguments. what do you think?

**Review by sebbycake — APPROVED** [review]
LGTM

**Pull request merged**